Update: add fixer for `sort-imports`

not-an-aardvark opened the pull request:
**What is the purpose of this pull request? (put an "X" next to item)**

[ ] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/rule-change-proposal.md))
[x] Add autofixing to a rule
[ ] Add a CLI option
[ ] Add something to the core
[ ] Other, please explain:

<!--
    If the item you've checked above has a template, please paste the template questions below and answer them. (If this pull request is addressing an issue, you can just paste a link to the issue here instead.)
-->

<!--
    Please ensure your pull request is ready:

    - Read the pull request guide (http://eslint.org/docs/developer-guide/contributing/pull-requests)
    - Include tests for this change
    - Update documentation for this change (if appropriate)
-->

<!--
    The following is required for all pull requests:
-->

**What changes did you make? (Give an overview)**

This adds a fixer for [`sort-imports`](http://eslint.org/docs/rules/sort-imports).

It only sorts `ImportSpecifiers`, and does not sort multiple import statements. Sorting multiple import statements would cause the evaluation order of the imported files to change.

This also makes a slight change to the `sort-imports` rule; it reports at most one specifier from a given import statement. This allows the entire list of specifiers to be sorted as a whole. For example:

```js
import {foo, baz, bar} from 'qux.js';

// Previously, both 'baz' and 'bar' would have been reported. Now, only 'baz' will be reported.
// However, this statement will be correctly autofixed to:

import {bar, baz, foo} from 'qux.js';
```

**Is there anything you'd like reviewers to focus on?**

Nothing in particular.

nzakas commented on tests/lib/rules/sort-imports.js:
```diff
@@ -236,23 +236,37 @@ ruleTester.run("sort-imports", rule, {
+        },
+        {
+            code: "import /* a */ { /* b */ foo /* c */ , /* d */ bar /* e */ , /* f */ baz /* g */ , /* h */ } /* i */ from 'qux.js';",
+            output: "import /* a */ { /* b */ bar /* c */ , /* d */ baz /* e */ , /* f */ foo /* g */ , /* h */ } /* i */ from 'qux.js';",
```
I don't think we should be attempting to autofix if there are comments present. Chances are that the comments are somehow related to the bindings being imported, so keeping them in place while switching the order of the identifiers is confusing.

not-an-aardvark commented:
That's a good point about comments; I hadn't thought of that.

* Added a test with linebreaks
* Updated the fixer to not fix cases where the imported bindings have comments between them, and added tests to verify this.
